test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs

```diff
@@ -86,11 +86,12 @@ private static XDocument GetEmptyConfig()
 
             var config = GetOrAddSection(doc, "config");
             var packageSources = GetOrAddSection(doc, "packageSources");
+            var packageSourceMapping = GetOrAddSection(doc, "packageSourceMapping");
```

Comment:
I think this is a bad merge. 

Things are duplicated now. 

I'd recommended squashing *all your commits* into one before running rebase, with `git reset --soft HEAD~<#OfYourCommits>`. 

That way you hit fewer issues when rebasing.